apps/server/src/controllers/integrationController.ts

```diff
@@ -48,62 +48,39 @@ const actionHandlers: Record<string, ActionHandler> = {
     return { payload: updatedEvent };
   },
   /* Message Service */
-  'set-timer-message': (payload) => {
-    if (!isPartialTimerMessage(payload)) {
-      throw new Error('Payload is not a valid timer message');
+  //TODO: maybe coerce the boolean values
+  message: (payload) => {
+    if (payload && typeof payload === 'object') {
+      if ('timer' in payload) {
+        if (!isPartialTimerMessage(payload)) {
+          throw new Error('Payload is not a valid timer message');
```

Comment:
instead of maintaining so many error types, should we have a single
`throw new Error('Payload is not valid');` ?